src/tests/inpty.cc

```diff
@@ -0,0 +1,104 @@
+/*
```

Comment:
Might it be better to do this in Perl?  It'd be much simpler, and independent from configure and Mosh utility code.